src/main/kotlin/stringcalculator/StringCalculator.kt

```diff
@@ -0,0 +1,36 @@
+package stringcalculator
+
+import stringcalculator.constants.ErrorType.INVALID_INPUT_BLANK
+import stringcalculator.constants.ErrorType.INVALID_INPUT_NULL
+import stringcalculator.enums.Operator
+import java.util.*
+
+class StringCalculator {
+    fun calculate(expression: String?): Double {
+        val splits = expression?.split(DELIMITER)
+            ?: throw IllegalArgumentException(INVALID_INPUT_NULL.message)
```

Comment:
Elvis 연산자를 사용하셨네요 👍 
그런데, Null이 아닌 다른 기타 값들을 제공하거나 하위로직을 단순히 실행 안시키고 반환할게 아니라면, 
예외 제한 블럭을 통해 유효성 검증을 하는 방식으로 하면 어떨까요? 

해당 방식들을 통해 스마트캐스팅 기능도 제공이 되기 때문에 좀 더 가독성과 편이성을 높힐 수 있어요.
```suggestion
        require(!expression.isNullOrBlank()){ 
            INVALID_INPUT_NULL.message
        }
        
        //이 이후로 expression은 not null임이 스마트 캐스트 되어 보장됩니다.
```